networking/proxies/configuring-cluster-wide-proxy.adoc

```diff
@@ -0,0 +1,32 @@
+include::modules/common-attributes.adoc[]
+ifdef::openshift-dedicated[]
+include::modules/attributes-openshift-dedicated.adoc[]
+endif::[]
+:context: cluster-wide-proxy-configuration
+[id="cluster-wide-proxy-configuration"]
+= Configuring a cluster-wide proxy during installation
+
+toc::[]
+
+Cluster-wide proxies may be configured during cluster creation.
+
+.Prerequisites
+
+This guide assumes that the following prerequisites are met:
+
+* You are the cluster owner.
+* Your account has sufficient privileges.
+* You must have a Customer Cloud Subscription (CCS) cluster with a VPC that the proxy can access.
```

Comment:
"CCS" is not really a concept for ROSA, only OSD. ROSA is by definition CCS - I think mentioning it in the ROSA documentation might because confusion for ROSA customers thinking that they need to obtain some separate subscription to use this feature.

So if there is a way to only have this reflected in the OSD docs that would be great.